packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -908,8 +909,17 @@ export default class TradeStore extends BaseStore {
                             const is_digit_contract = isDigitContractType(category?.toUpperCase() ?? '');
                             const is_multiplier = isMultiplierContract(category);
                             const contract_type = category?.toUpperCase();
-
-                            if ((window as any).hj) (window as any).hj('event', `placed_${category}_trade`);
+                            const is_call = category === 'Call';
+                            const is_put = category === 'Put';
+                            const is_high_low = isHighLow({ shortcode_info: extractInfoFromShortcode(shortcode) });
+                            const higher_lower_contact = is_call ? 'higher' : 'lower';
```

Comment:
Can we minimize the ternary operator ? By maybe initializing with default value and checking it in one conditional operator?
maybe something like 
let higher_lower_contact='lower';
let rise_fall_contract='fall';
if(is_call){
higher_lower_contact='higher';
rise_fall_contract='rise';
}

Reply:
done